spec/features/variable_drafts/create_variable_draft_from_variable_spec.rb

```diff
@@ -1,5 +1,5 @@
 describe 'Create new draft from variable', reset_provider: true do
-  context 'when editing a published variable' do
+  context 'when making a draft of a published variable' do
```

Comment:
super minor (and you can ignore if you disagree), but _if_ you end up making other changes, I think 'from' is more appropriate than 'of'